daemon/daemon_main.go

```diff
@@ -615,6 +615,9 @@ func init() {
 	flags.String(option.LB, "", "Enables load balancer mode where load balancer bpf program is attached to the given interface")
 	option.BindEnv(option.LB)
 
+	flags.Bool(option.EnableNodePort, false, "Enable NodePort type services by Cilium")
```

Comment:
AFAICT We can't support BPF-based nodeport on all kernel versions that Cilium supports, so that may play into the "should we set it by default" question.

Reply:
We already enable / disable certain features based on the kernel versions.

Reply:
Yes, we can flip the switch once we have everything integrated, and when we detect (probes, on TODO) that kernel fully supports it.

Reply:
Unless we cannot get this from k8s, I'll add the range parameter, makes sense.